.rubocop_todo.yml

```diff
@@ -0,0 +1,405 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2018-09-27 14:16:34 +0100 using RuboCop version 0.49.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 3
+# Cop supports --auto-correct.
+Layout/EmptyLinesAroundExceptionHandlingKeywords:
```

Comment:
Is it worth running rubocop with -a to get a lot of these layout/whitespace etc. fixes for 'free'?

Or is that for a seperate commit/branch?

Reply:
I'll wait untill we have the acceptance tests running before I let rubocop loose on this:P